clients/da-vinci-client/src/main/java/com/linkedin/davinci/store/rocksdb/RocksDBSstFileWriter.java

```diff
@@ -106,6 +115,7 @@ public RocksDBSstFileWriter(
     this.envOptions = envOptions;
     this.options = options;
     this.fullPathForTempSSTFileDir = fullPathForTempSSTFileDir;
+    this.fullPathForPartitionDBSnapshot = RocksDBUtils.composeSnapshotDir(dbDir, storeName, partitionId);
```

Comment:
Plz see my first comment. If this feature is not enabled, we don't need to create the path for this store.